tokenizer/lib/spacy/lang/mapper.py

```diff
@@ -0,0 +1,169 @@
+import importlib
+
+LANGUAGE = {
```

Comment:
I think, that it would be more flexible to place such long lists to separate json files.

Reply:
probably true :-) I will enter an item for that.